pandas/tests/apply/test_series_apply.py

```diff
@@ -640,7 +640,7 @@ def test_map_abc_mapping_with_missing(non_dict_mapping_subclass):
     # https://github.com/pandas-dev/pandas/issues/29733
     # Check collections.abc.Mapping support as mapper for Series.map
     class NonDictMappingWithMissing(non_dict_mapping_subclass):
-        def __missing__(key):
```

Comment:
this looks like it should fail in the status quo.  does this change fix anything?

Reply:
Sorry for the delay, @jbrockmendel ! In this case, for example, it doesn't fail because __missing__ is not directly called. Essentially these changes don't fix anything but code readability and correctness.